### DISCUSSION TITLE
Currency units for Ethereum

### DISCUSSION THREAD

**Discussion by optimisticben** [grafana/grafana#38984] [Dashboards, Panels & Visualizations]
We're making use of Grafana to visualize Ethereum network metrics and it would be useful to have the project's native units available in Grafana.

Specifically:
- ETH (the native unit of the network)
- gwei (an ETH denomination used for gas calculations)
- wei (the smallest ETH denomination)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
